fml/BUILD.gn

```diff
@@ -365,7 +362,10 @@ if (enable_unittests) {
       "time/time_unittest.cc",
     ]
 
-    if (is_mac) {
+    if (is_mac || is_ios) {
```

Comment:
Fix: making consistent with the associated conditional in the target above that this one is testing.